README.rst

```diff
@@ -392,14 +392,40 @@ More Information
 Getting Involved
 ----------------
 
-- `Forum`_: For discussions about development, questions about usage, and feature requests.
-- `GitHub Issues`_: For reporting bugs.
-- `Twitter`_: Follow updates on Twitter.
-- `Slack`_: Join our Slack channel.
-- `Meetup Group`_: Join our meetup group.
-- `StackOverflow`_: For questions about how to use Ray.
-
-.. _`Forum`: https://discuss.ray.io/
+.. list-table::
+   :widths: 25 50 25 25
+   :header-rows: 1
+
+   * - Platform
+     - Purpose
+     - Estimated Response Time
+     - Support Level
+   * - `Discourse Forum`_
+     - For discussions about development and questions about usage.
+     - < 2 day
```

Comment:
How about let's make this <1 day? 

Generally we're pretty good at this... and redirecting more traffic to Discourse will then further validate investment in this area!

Reply:
SGTM! Making the change now to `< 1 day` for Discourse forum responses. 😄